core/src/main/scala/org/apache/spark/deploy/worker/CommandUtils.scala

```diff
@@ -117,4 +114,15 @@ object CommandUtils extends Logging {
       }
     }.start()
   }
+
+  val SystemPropertyPattern = """^-D(.+?)=(.*)$""".r
```

Comment:
nit: either ALL_CAPS or camelCase. We don't declare variables with names like that in Spark.